BankSDK/GiniBankSDK/Sources/GiniBankSDK/Core/GiniBankConfiguration.swift

```diff
@@ -450,9 +450,7 @@ public final class GiniBankConfiguration: NSObject {
 		giniBankConfiguration.onboardingAlignCornersIllustrationAdapter = configuration.onboardingAlignCornersIllustrationAdapter
         
         giniBankConfiguration.entryPoint = configuration.entryPoint
-
-		giniBankConfiguration.debugModeOn = configuration.debugModeOn
```

Comment:
I will check to see if removing was a mistake.

Reply:
I've added them back since I removed them by mistake when fixing merge conflicts. I've checked with the previous pr where I've added them.